src/virtualenv/create/via_global_ref/builtin/python2/site.py

```diff
@@ -60,6 +66,10 @@ def load_host_site():
 sep = "\\" if sys.platform == "win32" else "/"  # no os module here yet - poor mans version
 
 
+def dir_starts_with(directory, prefix):
```

Comment:
this is out of order, usually, functions are defined after they are called

Reply:
so they read top down

Reply:
You should invest in some automated checking tools. Bothering contributors with such trivial details on each review is kinda frustrating.

Reply:
That's why we have pre commit. I haven't got a tool for checking this yet though.

Reply:
That being said feel free to take this as far as you're willing to. I'll pick up and clean things up once you're done, and ship it. As a maintainer, I feel like I need to guide the code to have some unified style and ensure PRs don't just fix issues but leave the codebase in a maintainable state. Hope you understand.

Reply:
You need a CI check, not (just) pre-commit.

If your plan is to manually guide people to your style and pick up behind people, you're going to be exhausted before long and even might lose contributors in the process. Which is not something I wish you!

Reply:
As I said I'm not aware of a tool that does this at the moment. Pre commit is run part of the ci. So a pre commit test is a ci test.